src/components/FavouriteStay/FavoriteStay.css

```diff
@@ -0,0 +1,82 @@
+/* rule for svg*/
+/* style="display: block; height: 8px; width: 8px; fill: currentcolor;" */
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
+
+.fav-stay-container {
+    border-radius: 12px;
+    border: 1px solid #DDDDDD;
+    align-items: center;
+    max-width: 741px;
+}
+
+.fav-stay-grid-container {
+    display: grid;
+    grid-template-columns: 1fr auto 1fr 1px 1.3fr auto 1.3fr 1px 1.3fr auto 1.3fr;
+    padding: 20px 0;
+}
+
+.host-badge-container {
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    gap: 2px;
+}
+
+.host-badge-left, .host-badge-right {
+    width: auto;
+    height: 32px;
+   
+
+}
+
+.host-badge-text {
+    padding-right: 4px 4px;
+    font-weight: 500;
+    font-size: 1.4rem; 
+    font-family: 'Montserrat', sans-serif; 
```

Comment:
I think you can remove this line and let the font family to inherit from its parent component. When I remove it it looks like this.
![image](https://github.com/user-attachments/assets/ecbeecef-e6f3-4cb6-9314-34f1b637794a)

Reply:
I adjusted the image and the requirements to match the updated actual website look, which is different from what is on the planning board.